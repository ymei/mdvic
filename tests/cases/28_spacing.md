Inline: $\sinx$, $\cos(y)$, $a\cdotb$, $A\subseteqB$, $x\rightarrow y$.

