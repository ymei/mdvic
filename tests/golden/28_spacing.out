Inline: sin x, cos(y), a · b, A ⊆ B, x →
 y.
